chat-plugins/info.js

```diff
@@ -627,9 +627,10 @@ exports.commands = {
 		if (!this.runBroadcast()) return;
 		target = target.trim();
 		let mod = target.split(',');
+		let pokemon = mod[0];
 		mod = Dex.mod(toId(mod[mod.length - 1])) || Dex;
 		let targets = target.split(/ ?[,/ ] ?/);
```

Comment:
What is this code even trying to do?

Reply:
Before the fix it would try to use the first word (after splitting on space) as pokémon name (e.g. ``/weak tapu bulu`` would search for ``tapu``, not finding it); after the fix, it tries the first "piece" after splitting on comma, so that ``/weak tapu bulu`` works again. I had it use the whole string as pokémon name, but then it would break on older generations (e.g. ``/weak togekiss, gen4`` would search for ``togekiss, gen4``) so I had it get the first result of the split (which is the whole string if it can't split it) and use that as pokémon name to fix that issue.

Reply:
Why first word and not first phrase split by `,`?

Reply:
That's what the fix does, searches the first phrase split by ``, ``.  I don't know why it tried to search the first word.

Reply:
So why does this line still exist?

Reply:
That's why I asked what this line is doing - if you're still keeping it around, I'm guessing it's used for something else?

Reply:
I think it's because the pokemon is guaranteed to be separated by a `,` (hence the `mod` variable), but types aren't (currently things like `/weak steel grass`, without any commas, works), so you split it by either space or comma for the types

Reply:
I would manually split the types by ` ` if there's only one, then. That seems the simplest solution.